packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx

```diff
@@ -178,6 +178,11 @@ export const ChatMessageListItem = (props: ChatMessageListItemProps) => {
     )
   }
 
+  const unfurlStyles = {
```

Comment:
nit: can call cn here and set it to eg `unfurlClassNames` and pass to className directly below